specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json

```diff
@@ -535,6 +535,14 @@
           "description": "managed identities for the Container App to interact with other Azure services without maintaining any secrets or credentials in code.",
           "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
         },
+        "managedBy": {
```

Comment:
Can this be null by default so it will be a breaking change?

Reply:
Yes, this is an optional property